Riot/Modules/Common/Cells/GroupedTableViewCell.swift

```diff
@@ -0,0 +1,47 @@
+// 
+// Copyright 2020 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+/// This can be used in grouped table views to hide section separators for a specific cell.
+class GroupedTableViewCell: UITableViewCell {
```

Comment:
To avoid to make some UITableViewCell inherit from `GroupedTableViewCell` and hack the group style we can use a protocol `TopSeparatorDisplayable`, `BottomSeparatorDisplayable` that expose separator outlet and embed separators directly into cells when needed. It will avoid to make this fragile hack on the view hierarchy.

Reply:
Updated, actually GroupedTableViewCell removed